### PR TITLE
vUART: change S5 vUART resource

### PR DESCRIPTION
**Pull request by YuanXin-Intel**
This patch is to change the vUART resource occupied by S5 function between Service VM and guest VM to avoid the standard UART port conflict when legacy UART passthrough to guest VM.

Tracked-On: #8622

Reviewed-by: Junjie Mao <junjie.mao@intel.com>
Reviewed-by: Jian Jun Chen <jian.jun.chen@intel.com>

### CODE REVIEW

File: misc/config_tools/static_allocators/s5_vuart.py [hunk 1]
```diff
@@ -49,23 +51,24 @@ def get_console_vuart_port(scenario_etree, vm_name):
...
-
-        global service_port_list
-        service_port_list = list(set(service_port_list) - set(tmp_list))
-        service_port_list.sort()
-        port = hex(service_port_list[0])
-        service_port_list.remove(service_port_list[0])
-        return str(port).upper()
-    else:
-        return "0x2F8"
+        vm_id = int(get_node(f"//vm[load_order = 'SERVICE_VM']/@id", scenario_etree))
```
Comment (junjiemao1) [inline misc/config_tools/static_allocators/s5_vuart.py:64]: Do we still need this step? I think VM names are already unique can identify any VM regardless of its type.

Comment (YuanXin-Intel) [reply misc/config_tools/static_allocators/s5_vuart.py]: Yes, this step is necessary for Service VM to obtain vm_id

**Comment by junjiemao1** [comment]
I'm fine for the configuration side. For the device model changes @chejianj can comment.

**Review by chejianj — APPROVED** [review]
LGTM for the device model part

**Review by NanlinXie — APPROVED** [review]

**Comment by yuhuanX** [comment]
start to run premerge test

**Pull request merged**